runtime/io-stmt.cpp

```diff
@@ -7,138 +7,77 @@
 //===----------------------------------------------------------------------===//
 
 #include "io-stmt.h"
+#include "connection.h"
+#include "format.h"
 #include "memory.h"
+#include "tools.h"
 #include "unit.h"
 #include <algorithm>
 #include <cstring>
+#include <endian.h>
```

Comment:
Is this standard? I can't find it on my macbook.

Reply:
The build environment on "mojave" could find it; but no, it's not standard.  I'll avoid using it.